### PR TITLE
Implement Parse method

### PR DESCRIPTION
**Pull request by NavinSeth**

### CODE REVIEW

**Comment by NavinSeth** [comment]
This is still a WIP, but feel free to take a sneak peak and lmk what you think so far.

File: src/helpers.ts [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+import moment from 'moment';
+
+export function getElementValue(
+  elementId: string,
+  barcodeDataLine: string
+): string {
+  return trim(barcodeDataLine).replace(elementId, '');
+}
+
+export function cleanLicenseDate(cleanedLine: string) {
```
Comment (NavinSeth) [inline src/helpers.ts:11]: We probably won't need this, since we aren't doing any date conversion/formatting

File: src/helpers.ts [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  let date: moment.Moment = moment(cleanedLine, 'MMDDYYYY'); // this is the accepted format in the AAVAA
+  if (!date.isValid()) {
+    // some states format it this way, I don't know why
+    // there won't be any collisions because this license format will always start with 19 or 20
```
Comment (robotmlg) [inline src/helpers.ts:15]: nice

**Review by robotmlg — APPROVED** [review]

**Comment by NavinSeth** [comment]
Closing for now but will come back when I have more time to work on this.

**Pull request closed without merging**